wiki/UI-component/Video.md

```diff
@@ -39,6 +39,147 @@ The [[Video]] component utilizes the embeddable Kaltura player <sup>[[r2](#resou
 
 <hr width="40%" />
 
+## 1. Data & content
+
+The video component intakes a Kaltura `videoId` and fetches the following data to be rendered on page as content. All of these content can be customized except the video duration.
+
+See more details at [Kaltura Player API](http://player.kaltura.com/docs/api).
+
+| Data name | Type | Allows override |
+| --------- | ---- | --------------- |
+| Name | Text | Yes |
+| Description | Text | Yes |
+| Duration | Number | No |
+| Poster image | Image | Yes |
+
+## 2. Rendering & behavior
+
+Video is used in the following components. Depending on the visual design and content requirement of the final component rendered, the above data is used differently.
+
+### Video
+
+This is the default rendering of video. It is used for embedding a video directly on page or inline.
+
+![video](https://user-images.githubusercontent.com/15144993/123469683-5cf69380-d5c1-11eb-844f-39e26678c961.png)
+
+| ID  | Data name     | Field |
+| --- | ------------- | ----- |
+| 1   | Name   | Caption |
+| 2   | Duration | appended after caption in parenthesis |
+| 3   | Description | Poster image alt tag |
+| 4   | Poster image | Image |
+
+Interactions
+
+| ID  | Element name | Event | Behavior | Notes |
+| --- | ------------ | ----- | -------- | ----- |
+| 1   | Poster image | On Click or On Enter key of link | Poster image will be replaced by Kaltura video player that will begin playing <br /> Alternative option to open in LightBoxMediaViewer. | Poster image obeys aspect ratios: 16x9, 9x16, 2x1, 1x2, 4x3, 3x4, 1x1 |
```

Comment:
Scoping it all back. This is where your functional specs for this component begin. Everything around this belongs in something else.